### PR TITLE
Use demo-snippet in demos.

### PR DESCRIPTION
**Pull request by bicknellr**
screen shot 2016-01-22 at 12 26 16
screen shot 2016-01-22 at 12 26 31
screen shot 2016-01-22 at 12 26 52
screen shot 2016-01-22 at 12 27 04

### CODE REVIEW

**Comment by bicknellr** [comment]
I fixed up this one since it had the same problems mentioned in your comments on PolymerElements/iron-image#61.

File: demo/index.html [hunk 1]
```diff
@@ -19,163 +19,233 @@
...
+          </div>
+        </template>
+      </demo-snippet>
+
+      <h3><code>align-bottom</code> Attribute</h3>
+      <demo-snippet>
+        <template>
+          <style is="custom-style">
+            paper-tabs[align-bottom] {
+              box-shadow: 0px -2px 6px rgba(0, 0, 0, 0.15);
```
Comment (notwaldorf) [inline demo/index.html]: maybe apply a shadow form `paper-styles/shadow.html`?

**Comment by notwaldorf** [comment]
LGTM! My only comment would be to use full sentences as the headers describing what that little snippet does. 🎨

File: demo/index.html (hunk 1 above)
Comment (bicknellr) [inline demo/index.html]: Removed - the tabs don't actually include any box-shadow by default.

**Pull request merged**